src/cf-forms/usage.md

```diff
@@ -110,16 +86,34 @@ See the 'Form icons' section below for guidance on adding icons to states.
        title="Test input">
 ```
 
-### Input icons
-
-Form input icons add small positioning tweaks to Capital Framework icons.
+### Inline Form Validation
 
-<input type="text" value="" title="Test input">
-<span class="cf-form_input-icon cf-icon cf-icon-email"></span>
+<div class="m-field m-field__error">
+    <label class="a-label__heading" for="form-input-error">Label</label>
+    <input class="a-text-input a-text-input__error"
+           type="text"
+           value="Invalid input"
+           id="form-input-error"
+           aria-describedby="form-input-error_message">
+    <div class="a-error-message" id="form-input-error_message" role="alert">
```

Comment:
What do you think about calling this `a-field-alert a-field-alert__error` so that it's extensible? That would also line it up with the naming in the Design Manual fork you've referenced.